http/get_multipart/python/client/simple_client.py

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Simple HTTP client parsing a multipart/mixed response.
+
+This client parses the multipart response produced by server/server.py
+by using the multipart message parser from the Python email module.
+
+This module puts the entire message in memory and seems to spend a lot
+of time looking for part delimiter and encoding/decoding the parts.
+
+The overhead of multipart/mixed parsing is 85% on my machine and after
+the ~1GB Arrow Stream message is fully in memory, it takes only 0.06%
+of the total execution time to parse it.
+"""
```

Comment:
Could you please put this in a separate `README.md` file?

(like https://github.com/apache/arrow-experiments/blob/main/http/get_simple/python/client/README.md)